.github/workflows/verify.yml

```diff
@@ -44,9 +44,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2.4.0
       - name: deps
-        run: sudo apt-get update && sudo apt-get install -yq libpcsclite-dev
+        run: sudo apt-get update
```

Comment:
Do we even need this?

Reply:
I don't believe we ever needed `libpcsclite` for the linter

Reply:
I meant we don't need this `sudo apt-get update`

Reply:
for golangci-lint we dont need apt udpate/upgrades/install

Reply:
@cpanato @naveensrinivasan removed, PTAL